src/app/meshes/views/MeshView.vue

```diff
@@ -35,31 +39,28 @@
 </template>
 
 <script lang="ts" setup>
-import { KTabs } from '@kong/kongponents'
+import { KTabs, Tab } from '@kong/kongponents'
 import { useRouter } from 'vue-router'
 
 import AppView from '@/app/application/components/app-view/AppView.vue'
 import RouteView from '@/app/application/components/route-view/RouteView.vue'
 import { useI18n } from '@/utilities'
 
 const { t } = useI18n()
-const router = useRouter()
 
-const meshRoutes = router.getRoutes().find((route) => route.name === 'mesh-abstract-view')?.children ?? []
-const items = meshRoutes.map((item) => {
-  if (typeof item.name === 'undefined') {
-    const route = item.children?.[0]
-    const name = String(route?.name)
+const router = useRouter()
+const c = router.getRoutes().find((route) => route.name === 'mesh-abstract-view')?.children ?? []
 
-    return {
-      title: t(`meshes.routes.item.navigation.${name}`),
-      hash: name,
-    }
-  }
+const meshRoutes = c[0].children ?? []
+const data: Map<string, {module: string | undefined}> = new Map()
+const items: Tab[] = meshRoutes.map((item) => {
   const name = String(item.name)
+  data.set(`#${name}`, {
+    module: item?.meta?.module,
+  })
   return {
     title: t(`meshes.routes.item.navigation.${name}`),
-    hash: name,
+    hash: `#${name}`,
```

Comment:
Eventually I should be able to remove all this code from this file and just end up with the KTabs.